crates/ty_python_semantic/resources/mdtest/call/methods.md

```diff
@@ -458,10 +458,14 @@ class C:
     def f2(cls: type[C], x: int) -> str:
         return "a"
 
-reveal_type(C.f1(1))  # revealed: str
-reveal_type(C().f1(1))  # revealed: str
-reveal_type(C.f2(1))  # revealed: str
-reveal_type(C().f2(1))  # revealed: str
+# TODO: revealed: str
+reveal_type(C.f1(1))  # revealed: str | Divergent
+# TODO: revealed: str
+reveal_type(C().f1(1))  # revealed: str | Divergent
+# TODO: revealed: str
+reveal_type(C.f2(1))  # revealed: str | Divergent
+# TODO: revealed: str
+reveal_type(C().f2(1))  # revealed: str | Divergent
```

Comment:
This definitely seems likely to impact real projects, suggesting that https://github.com/astral-sh/ty/issues/1732 is a pre-beta follow-up.